packages/editor-ui/src/views/helpers.ts

```diff
@@ -0,0 +1,47 @@
+import { INodeUi } from "@/Interface";
+
+export const getLeftmostTopNode = (nodes: INodeUi[]): INodeUi => {
+	let leftmostTop = nodes[0];
+
+	nodes.forEach(node => {
+		if (node.position[0] < leftmostTop.position[0]) {
+			leftmostTop = node;
+		}
+
+		if (node.position[0] < leftmostTop.position[0] && node.position[1] < leftmostTop.position[1]) {
+			leftmostTop = node;
+		}
```

Comment:
we are doing the same `node.position[0] < leftmostTop.position[0]` check twice and then doing the same `leftmostTop = node` setting, any way to combine or simplify this?